jest.config.js

```diff
@@ -126,7 +126,7 @@ module.exports = {
   // snapshotSerializers: [],
 
   // The test environment that will be used for testing
-  testEnvironment: "node",
+  testEnvironment: "jsdom",
```

Comment:
Were you able to get supertest to work with jsdom? I got an error saying something about it being incompatible with jsdom

Reply:
No! Same here. Take a look at the top of my server.spec.js file. There's a little comment block that lets you override the testEnvironment from what's in the config and set it to a different environment just for that file's tests. I don't remember where I read about it, it was in one of those articles or possibly the jest docs. Super weird.